### PR TITLE
Shop Booking

### PR DESCRIPTION
**Pull request by mohokh67**
### Description

This Pull Request introduces shop booking system
### List of tasks

- [x] Branch has been updated with `Master`
- [x] Manually tested the new functionality
- [ ] Added relevant tests
- [ ] Updated relevant documentation
- [ ] Demo the functionality to a team member

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,9 @@
             "pre-commit": "lerna run --concurrency 1 --stream precommit --since HEAD --exclude-dependents"
         }
     },
-    "dependencies": {},
+    "dependencies": {
```
Comment (mohokh67) [inline package.json:24]: It will fi the problem with `environment variables` on Windows OS

File: packages/serverless/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         "precommit": "lint-staged",
         "prettier": "prettier --write src/**/*.ts",
         "start": "yarn clean && sls offline start",
-        "test": "LOG_LEVEL=fatal jest --detectOpenHandles --colors",
+        "test": "cross-env LOG_LEVEL=fatal jest --detectOpenHandles --colors",
```
Comment (mohokh67) [inline packages/serverless/package.json:18]: this is how we use `cross-env`

File: packages/shop/src/components/Main/Booking/Booking.js [hunk 3]
```diff
@@ -1,28 +1,52 @@
...
+            <Days show={availableDaysVisibility} chooseDate={chooseDate} />
+            <br />
+            <Sessions show={availableSessionsVisibility} chooseSession={chooseSession} />
+            <br />
+            <AllStaff show={availableStaffVisibility} chooseStaff={chooseStaff} />
             <br />
-            <WorkingBarbers show={workingHoursVisibility} selectedDate={selectedDate} />
-        </div>
+            <Details
+                date={selectedDate}
```
Comment (mohokh67) [inline packages/shop/src/components/Main/Booking/Booking.js:44]: @amirshd22 once we add `Redux` then there should be less code to handle

**Review by amirshd22 — APPROVED** [review]

**Review by MrMarius — COMMENTED** [review]
looks good

**Pull request merged**